.github/workflows/storage.yml

```diff
@@ -108,56 +101,28 @@ jobs:
     strategy:
       matrix:
         target: [ios, tvos, macos, watchos]
-        spec: [
-          'FirebaseStorage.podspec --test-specs=unit', # The integration tests need more set up.
-          'FirebaseStorageSwift.podspec --skip-tests' # No current unit tests.
-        ]
     steps:
     - uses: actions/checkout@v2
     - name: Setup Bundler
       run: scripts/setup_bundler.sh
     - name: Build and test
       run: |
-       scripts/third_party/travis/retry.sh scripts/pod_lib_lint.rb ${{ matrix.spec }} --platforms=${{ matrix.target }}
+       scripts/third_party/travis/retry.sh scripts/pod_lib_lint.rb FirebaseStorage.podspec --test-specs=unit --platforms=${{ matrix.target }}
 
   storage-cron-only:
     # Don't run on private repo.
-    if: github.event_name == 'schedule' && github.repository == 'Firebase/firebase-ios-sdk'
+#    if: github.event_name == 'schedule' && github.repository == 'Firebase/firebase-ios-sdk'
```

Comment:
should this still be commented out?

Reply:
Yes - for this branch while we work through the Storage changes for v9.  We'll need a complete review when we merge back to master.